src/utils/graph_factory.hpp

```diff
@@ -392,6 +426,14 @@ GraphFactory::convert_to(const LIBSBML_CPP_NAMESPACE::Model& model, G& g) const
       }
     }
   }
+  #if !defined(WCS_HAS_EXPRTK)
+  // delete .o and .so files
+  const auto fn = std::filesystem::path(library_file);
+  std::string stem = fn.stem();
+  std::string lib_filename1 = stem + ".o";
+  std::string system1 = " rm " + library_file + " " + lib_filename1;
+  system(system1.c_str());
```

Comment:
This should probably rely on the function defined in utils/file.hpp instead of directly using std::filesystem such that this still compiles when std::filesystem is not available by switching to boost::filesystem.